README_dev.md

```diff
@@ -17,6 +18,14 @@ This workspace provide two types of scripts:
 
     additionally you can run a script in all workspaces by `npm run -ws --if-present <scriptname>`
 
+## Linting
+
+-   Typescript
+    -   each package provides its own typescript linting script. You can run it by `npm run lint:ts` from the package directory or by `npm run -w packages/<package> lint` from the root of the project
+    -   the root project also provides a script that lints all packages. You can run it by `npm run lint:ts` from the root of the project
+-   Prettier: simply run `npm run prettier` from the root of the project
+-   ESLint: simply run `npm run lint:eslint` from the root of the project
```

Comment:
This fails on my computer

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

Reply:
oh no

Reply:
fixed - wsl had not enough memory